ingeniamotion/fsoe_master/safety_functions.py

```diff
@@ -329,6 +346,45 @@ class SS1Function(SafetyFunction):
         uid="FSOE_SS1_ACTIVATE_SOUT_{i}", display_name="Activate SOUT"
     )
 
+    __FSOE_SAFE_INPUTS_MAP_ACTIVE_VALUE = 2
```

Comment:
shouldnt these vales come from the enum of reg, and maybe in the safeinputs function class

Reply:
I will create an enum in ingeniamotion. Get the enum from dictionary can be problematic, because values never changes, but some enums had changes in the labels. In addition, enum labels in the dictionary has spaces because are human-readable, but it is ugly for ingeniamotion.